libs/hpc-live/src/lib/model.ts

```diff
@@ -675,6 +675,123 @@ export class LiveModel implements Model {
           signal: params.signal,
         });
       },
+      getFlowsDownloadXLSX: (params) => {
+        const query = `query {
+          searchFlowsBatches${searchFlowsParams(params)} {
```

Comment:
We are copying the query from `searchFlows`, it would be a good idea to have a single definition of the fields we receive from `flows` to not forget to modify one of them

Reply:
Sure thing! I've declared it as a service 'readonly' property